catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/suggestion/LatLonCoordinateProcessor.java

```diff
@@ -14,36 +14,37 @@
 package org.codice.ddf.catalog.ui.query.suggestion;
 
 import java.util.ArrayList;
-import java.util.LinkedList;
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import org.codice.ddf.spatial.geocoding.Suggestion;
 
 /**
- * The {@link LatLonCoordinateProcessor} enables search suggestions for coordinate literals in a
- * variety of formats.
+ * The {@link LatLonCoordinateProcessor} enables search suggestions for Lat/Lon coordinate literals
+ * whenever a pair of decimal numbers is encountered within the expected bounds. The general policy
+ * is to be as forgiving as possible without sacrificing determinism.
  */
 public class LatLonCoordinateProcessor {
   private static final String REGEX_LAT_LON = "[-]?(([\\d]+(\\.[\\d]+)?)|(\\.[\\d]+))";
 
   private static final Pattern PATTERN_LAT_LON = Pattern.compile(REGEX_LAT_LON);
 
+  // This key tells the UI that the geo is on the suggestion itself
+  private static final String LITERAL_SUGGESTION_ID = "LITERAL1";
```

Comment:
Based on this comment, `LITERAL1` also signifies that it's lat/lon and not MGRS? Also if you want to signify the presence of a geo on a suggestion why not use a boolean? 

https://github.com/codice/ddf/pull/3901#discussion_r227495806

Reply:
`LITERAL1` as an ID says "I'm a literal suggestion, so I bring my own geo information", but only because I start with `LITERAL`. The `1` is strictly to keep IDs unique. It doesn't matter if Lat/Lon or MGRS get `1` or `2`. 

Per the `Suggestion` interface, we only have `id` and `name` to work with. Those are the guaranteed fields on a suggestion. Our impl adds more but the UI can't assume that. 

Updating the interface itself was agreed upon as out of scope for this work.

Reply:
If the `1` and `2` is strictly to keep IDs unique, then why not `LITERAL_LAT_LON` and `LITERAL_MGRS`?

Reply:
Can do 👍